client/packages/purchasing/src/purchase_order/DetailView/LineEdit/columns.ts

```diff
@@ -32,6 +62,9 @@ export const usePurchaseOrderLineEditColumns = ({
                 setter: (patch: Partial<DraftPurchaseOrderLine>) => {
                   updatePatch({ ...patch });
                 },
+                cellProps: {
+                  isDisabled: true, // Edited via number of packs for now
+                },
```

Comment:
Since this is always disabled, should probably remove the "setter" function -- it never gets used, the value is updated via the other fields)